components/post_view/post_info/post_info.jsx

```diff
@@ -134,6 +134,7 @@ export default class PostInfo extends React.PureComponent {
         };
 
         this.postHeaderRef = React.createRef();
+        this.dotMenu = React.createRef();
```

Comment:
In order to be conssistent with other PRs can you rename it like
```suggestion
        this.dotMenuRef = React.createRef();
```

Reply:
Ok!